openpype/hosts/maya/plugins/load/load_image.py

```diff
@@ -286,7 +286,7 @@ def _get_colorspace(self, context):
 
         path = get_representation_path_from_context(context)
         colorspace = get_imageio_colorspace_from_filepath(
```

Comment:
Actually wait.. is this function deprecated?

See https://github.com/ynput/OpenPype/blob/develop/openpype/pipeline/colorspace.py#L190-L193

Should we be using the replacement for it?

Reply:
Also just so you know - this logic **should** only be used when you're loading a file that has no colorspace data published along @moonyuet - is that the case for your failing case as well?

Reply:
> Also just so you know - this logic **should** only be used when you're loading a file that has no colorspace data published along @moonyuet - is that the case for your failing case as well?



I have replaced the deprecated function with the current function in the latest commit. In my case if it is from SP, it wont error out, but it is spotted by @LiborBatek and some error encountered when he loaded the textureset. The image below is his screencapture when he hit the bug.
<img width="806" alt="textureset_image" src="https://github.com/ynput/OpenPype/assets/64118225/5678ace1-256d-4084-9844-a1945810ebaa">

Reply:
Would be good to also double check whether by some bug that publish didn't include colorspace data or not. Maybe there was no color management enabled of course. :)